kpsewhich TEXMFROOT is not the correct value to use in our setup.

--- a/texk/texlive/linked_scripts/texlive/updmap.pl	2021-01-21 11:15:02.314074252 +1100
+++ b/texk/texlive/linked_scripts/texlive/updmap.pl	2021-01-21 11:15:53.365456336 +1100
@@ -19,7 +19,7 @@
 my $TEXMFROOT;
 BEGIN {
   $^W = 1;
-  $TEXMFROOT = `kpsewhich -var-value=TEXMFROOT`;
+  $TEXMFROOT = "/usr/share/texmf-dist";
   if ($?) {
     die "$0: kpsewhich -var-value=TEXMFROOT failed, aborting early.\n";
   }
